app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -89,5 +90,15 @@ def ordered_also_attends_cabinet
 
       sorted_ministers.map(&:content_id)
     end
+
+    def ordered_ministerial_departments
```

Comment:
it looks like this method returns content_ids so perhaps `ordered_ministerial_department_content_ids` would be a better name.

Reply:
Good idea.  I'll change the method names.